platform/linux-generic/odp_fdserver.c

```diff
@@ -647,6 +647,32 @@ int _odp_fdserver_init_global(void)
 	}
 
 	if (server_pid == 0) { /*child */
+		sigset_t sigset;
+		struct sigaction action;
```

Comment:
Parts of this struct may be left uninitialized. I would suggest memset(&action, 0, sizeof(action)) at some point before the sigaction call.

Reply:
Yes, thanks